cpp/memilio/mobility/metapopulation_mobility_instant.h

```diff
@@ -179,6 +206,20 @@ class MigrationParameters
     {
         m_coefficients = coeffs;
     }
+
+    /**
+     * @return the indices we want to save for the edge
+     */
+    const auto& get_save_indices() const
+    {
+        return m_save_indices;
+    }
+
+    const auto& get_save_indices()
+    {
+        return m_save_indices;
+    }
```

Comment:
This function is superfluous, or did you mean to make the return type non-const?

Reply:
The main idea for this function was to test whether the indices were set correctly. Currently used in test_graph.